### ISSUE TITLE
Fix L112.ghg_tgej_R_en_S_F_Yh not matching

### ISSUE THREAD

**Issue by bpbond** [JGCRI/gcamdata#569]
We mistakenly merged #545 that wasn't passing tests. In particular:

```
1. Failure: matches old data system output (@test_oldnew.R#115) ----------------
>round_df(olddata) not equivalent to round_df(newdata).
>Rows in x but not y: 2705. Rows in y but not x: 2705. 
>L112.ghg_tgej_R_en_S_F_Yh.csv doesn't match
```

Some initial investigation by @russellhz suggests that this is happening because of small–_very_ small–differences in `L101.in_EJ_R_en_Si_F_Yh` that then (probably because of the line 145 division in L112 chunk) produce larger differences in its output.

Right now this looks like a problem that @abigailsnyder and I have dealt with before: tiny differences getting propagated to large differences because very small numbers are being divided.

For now, I have opened #568 to disable L112 so `master` stops failing.

**Comment by russellhz** [comment]
Here is a better idea of what the issue is. On the left is gcamdata, the right is gcam-data-system-old. Starting at 1993, they are identical. In the earlier years (with data originating from `L101.in_EJ_R_en_Si_F_Yh`) the differences in input.emissions and energy are individually unimportant, but matter when they are divided to produce emfact. The NAs will become zeros in the next step, FYI
![screen shot 2017-07-28 at 1 45 45 pm](https://user-images.githubusercontent.com/25437326/28729756-8dd6f706-739b-11e7-8822-d229853d07a8.png)

**Comment by russellhz** [comment]
This problem actually originates in lines 220-225 of `chunk_LB1322.Fert.R`. The code looks right, but because the differences in numbers are so minuscule, errors occur. Below I show the difference in just the first step (line 220-221). These are side by side showing the `in_indenergy_netFert` and `in_indfeed_netFert` columns, which look identical between left (gcamdata) and right (old ds). However when I add an addition column (`in_indenergy_netFert + in_indfeed_netFert `), the results are different.

![screen shot 2017-07-29 at 7 13 16 am](https://user-images.githubusercontent.com/25437326/28744400-1d1b990c-742e-11e7-8492-38867cc0537c.png)

**Comment by bpbond** [comment]
Fixed in #603
